TINY-6009 Added variables to set toolbar backgrounds colors for inline and sticky toolbars (#6497)
tinymce/tinymce: modules/tinymce/changelog.txt

    Added the ability to resize `video` and `iframe` media elements #TINY-6229
    Added a new `font_css` setting for adding fonts to both the editor and the parent document #TINY-6199
    Added a new `ImageUploader` API to simplify uploading image data to the configured `images_upload_url` or `images_upload_handler` #TINY-4601

>     Added Oxide variable to define the container background color in fullscreen mode #TINY-6903

Reviewer: This wasn't merged correctly, so some changes @tylerkelly13 did have been lost.

- [ ] Fix changelog merge issues

Reviewer: I've fixed this, however @tylerkelly13 can you please review the new entry to make sure it looks okay?

Reviewer 2: @lnewson,

Added an Oxide variable for defining....

?

Reviewer: @tylerkelly13, it's multiple variables so that would be misleading.

Reviewer 2: Added Oxide variables .....

Reviewer: I'm confused? It already says that... are you looking at the above, as that's not what's currently in the changelog.

    Added a new `font_css` setting for adding fonts to both the editor and the parent document #TINY-6199
    Added a new `ImageUploader` API to simplify uploading image data to the configured `images_upload_url` or `images_upload_handler` #TINY-4601
    Added an Oxide variable to define the container background color in fullscreen mode #TINY-6903

>     Added Oxide variables to set toolbar backgrounds colors for inline and sticky toolbars #TINY-6009

Reviewer 2: @lnewson this make sense?
```suggestion
    Added Oxide variables for setting the toolbar background colors for inline and sticky toolbars #TINY-6009
```
?

Reviewer: LGTM :+1:
